guide/process/process-file.md

```diff
@@ -19,246 +19,41 @@ It is structured in the following way:
-    taskKey: taskY
-    inputs:
-      inputA: "Input1 to the task"
-      # or
-      inputB:
-        stepKey: taskX
-        key: taskZ
-  # ...
-```
-
-### Filter
-
-Apply one or multiple conditions on the previous step's outputs.
-
-All conditions should match to continue to the next step.
-
-<param-table :parameter="{
-  fields: [{
-    name: 'conditions',
-    description: 'Key-value map where the key references a data of the previous steps outputs and the value is the expected matching value.',
-    fullType: 'map&lt;key, string&gt;'
-  }]
-}" :types="{}" />
-
-#### Example
-```yaml
-key: erc20-notification
-steps:
-  # ...
-  - type: filter
-    conditions:
-      recipientID: "XXX"
-      contractAddress: "0x420167d87d35c3a249b32ef6225872fbd9ab85d2"
-  # ...
-```
-
-## Instance resolution
-
-[Triggers](#trigger) and [tasks](#task) must have a specific `instanceHash` but this can be resolved automatically by the compiler if it replaced by an object `instance` containing the following:
-
-<param-table :parameter="{
-  fields: [{
-    name: 'service',
-    description: 'Service\'s sid or hash to start. Cannot be used with &lt;code&gt;src&lt;/code&gt;.',
-    fullType: 'string'
-  }, {
-    name: 'src',
-    description: 'Path of the service to deploy and start. Local and remote path are supported. Cannot be used with &lt;code&gt;service&lt;/code&gt;.',
-    fullType: 'string'
-  }, {
-    name: 'env',
-    label: 'repeated',
-    description: 'List of environment variables to inject in the instance. Should respect the format: &lt;code&gt;VARIABLE=value&lt;/code&gt;.',
-    fullType: 'string'
-  }]
-}" :types="{}" />
-
-#### Example
-```yaml
-key: erc20-notification
-steps:
-  - type: trigger
-    instance:
-      src: https://github.com/mesg-foundation/service-ethereum-erc20
-      env:
-        - PROVIDER_ENDPOINT=$(env:PROVIDER_ENDPOINT)
-    eventKey: transfer
-  - type: task
-    instance:
-      service: "serviceX" # Start the service with sid or hash value `serviceX`
-      env:
-          - SENDGRID_API_KEY=$(env:SENDGRID_API_KEY)
-    taskKey: send
-  - type: task
-    instanceHash: "H74Qqq8nT5JZ9GSJmuSWLN5benWZPkUb5pYcvQLsoZX"
-    taskKey: task1
-  # ...
-```
-
-::: warning
-To support this feature, you need to compile the process with the [`--dev` flag](deployment.md#development-mode).
-:::
+A process has multiple steps, and each step indicates a specific action to do.
+- [**type: trigger**](./steps/trigger): that listen for a specific event or result.
+  - Should be the first step in the process.
+  - Only one trigger is possible per process.
+- [**type: task**](./steps/task): executes a specific task from a given instance of a service.
+  - You can create as many tasks as you want.
+  - A process should have at least one task.
+- [**type: filter**](./steps/filter): breaks the flow of your process if a condition doesn't match.
+  - You can create as many filters as you want.
+  - Filters are optional.
+
+
+<nav class="cards">
```

Comment:
It looks a lot easier for the navigation otherwise it's just small links in a list. Lets see if @EmmanuelDrouin can find something quickly otherwise I'm ok to remove it.

Reply:
OK done just added the new icons